src/libs/intents/localMethods.ts

```diff
@@ -30,7 +30,7 @@ export const asyncLogout = async (client?: CozyClient): Promise<null> => {
   await deleteKeychain()
   await clearCookies()
   await clearData()
-  RootNavigation.reset(routes.stack, { screen: 'welcome' })
+  RootNavigation.reset(routes.welcome, { screen: 'welcome' })
```

Comment:
note for later: `reset('foo', { screen : 'foo' })` looks weird, we should rewrite reset so the reset route is also automatically the first indexed route. Actually I don't know why it would be otherwise

Reply:
> we should rewrite reset so the reset route is also automatically the first indexed route

I'm not sure about this. In the NetService you use reset to go back to the callbackRoute that is not necessary the first indexed route

Also you can open the app with the first route being `home` but on logout you want to reset to the `welcome` route.
However i'm not sure why we need the `{ screen: ... }` param here.

Reply:
Well that's the point, with the new routing, reset() doesn't need two params, but just one, since we don't have two navigators (well we still do, but one has only one route so we can consider it as a route).

But anyway this is to be considered, it's not directly linked to your PR but the question is raised from the new routing